### PR TITLE
feat: broadcast screen

### LINKED ISSUE #265: UI: Broadcast Screen

### PR DESCRIPTION
**Pull request by reallybeard**
## Description

Added broadcast screen, reworked the components to be more modular to change the data more easily.

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?
- [x] Do all new and existing tests pass? Does the linter pass?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #265 

## Testing

Please outline all testing steps

1. Pull branch locally and run `yarn` to install new deps
2. etc...

## Screenshots (if applicable)
Screen Shot 2021-11-02 at 10 23 35 AM

### CODE REVIEW

File: src/context/EarnManagerProvider/components/BroadcastTx/BroadcastTx.tsx [hunk 1]
```diff
@@ -1,20 +1,51 @@
...
 import React from 'react'
+import { useTranslate } from 'react-polyglot'
+import { AssetToAsset, AssetToAssetProps } from 'components/AssetToAsset/AssetToAsset'
+import { SlideTransition } from 'components/SlideTransition'
 
-type Status = 'pending' | 'success' | 'fail'
+type Status =
+  | 'modals.broadcast.header.pending'
+  | 'modals.broadcast.header.success'
+  | 'modals.broadcast.header.error'
```
Comment (technojak) [inline src/context/EarnManagerProvider/components/BroadcastTx/BroadcastTx.tsx:10]: maybe we switch to an enum or const. I'll look at it more when wiring up

**Review by technojak — APPROVED** [review]

File: src/constants/FeatureFlagEnum.ts [hunk 2]
```diff
@@ -7,6 +7,6 @@ export type Flag = typeof FeatureFlagEnum[keyof typeof FeatureFlagEnum]
 
 export const FeatureFlagEnum = {
   Test: FlagValue.On,
-  Yearn: FlagValue.Off,
```
Comment (theoboldfrazier) [inline src/constants/FeatureFlagEnum.ts:10]: Do we want to keep this off?

**Review by theoboldfrazier — APPROVED** [review]
Approved with comment.

**Comment by cjthompson** [comment]
Can you add applicable screenshots to the description?

**Comment by reallybeard** [comment]
@cjthompson added screenshot

File: src/context/EarnManagerProvider/providers/yearn/components/YearnManager/YearnDeposit.tsx [hunk 3]
```diff
@@ -93,39 +96,133 @@ export const YearnDeposit = () => {
...
-              <Row.Label></Row.Label>
-              <Row.Value></Row.Value>
-            </Row>
+            <Stack spacing={6}>
+              <Row>
+                <Row.Label>
+                  <Text translation='modals.broadcast.transactionId' />
+                </Row.Label>
+                <Row.Value>
+                  <Link href='http://google.com' isExternal color='blue.500' fontWeight='bold'>
```
Comment (0xdef1cafe) [inline src/context/EarnManagerProvider/providers/yearn/components/YearnManager/YearnDeposit.tsx:182]: ?

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**